src/sync/app.hpp

```diff
@@ -111,7 +108,8 @@ class App {
     std::string m_app_route;
     std::string m_auth_route;
 
-    int m_request_timeout;
+    /// timeout in milliseconds, default is 60 seconds
+    uint64_t m_request_timeout = 60000;
```

Comment:
Can we make this `m_request_timeout_ms` for readability?

Reply:
done